recipes/dmgbuild/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "dmgbuild" %}
+{% set version = "1.6.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 7ced2603d684e29c22b4cd507d1e15a1907e91b86259924b8cfe480d80553b43
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  entry_points:
+    - dmgbuild = dmgbuild.__main__:main
+
+requirements:
+  host:
+    - python >=3.9
+    - pip
+    - setuptools
+  run:
+    - python >=3.9
+    - __osx                # [osx]
```

Comment:
I wonder if there is a linter bug here. This should be enough. @isuruf, am I missing something obvious here?

Reply:
@ocefpaf I found and followed what was made in https://github.com/conda-forge/applaunchservices-feedstock/blob/main/recipe/meta.yaml but it still fails.
There really must be an error in the linter.
One possibility is that the conda-forge.yml file is not being considered, as I renamed it to .yaml and it continues to give the same error.
Can you commit and then we can make any necessary adjustments to the feedstock?

Reply:
@ocefpaf Thanks!
https://github.com/conda-forge/dmgbuild-feedstock/pull/1